py/picca/delta_extraction/astronomical_objects/forest.py

```diff
@@ -289,15 +218,7 @@ def coadd(self, other):
                                           f"not happen. this.los_id={self.los_id}, "
                                           f"other.los_id={other.los_id}.")
 
-        if Forest.wave_solution == "log":
-            self.log_lambda = np.append(self.log_lambda, other.log_lambda)
-        elif Forest.wave_solution == "lin":
-            self.lambda_ = np.append(self.lambda_, other.lambda_)
-        else:
-            raise AstronomicalObjectError("Error in coadding Forest. "
-                                          "Class variable 'wave_solution' "
-                                          "must be either 'lin' or 'log'. "
-                                          f"Found: {Forest.wave_solution}")
+        self.log_lambda = np.append(self.log_lambda, other.log_lambda)
```

Comment:
Is numpy.append the right thing to use here? Is it ok if the log_lambda array is not sorted? It might be, just asking.

Reply:
It does work as expected as the rebin function at the end of coadd will take care of sorting everything into output wavelength bins in the end. So all that is really needed is having aligned arrays with merged self and other quantities

Reply:
Great. Thanks!